<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="GCTHW5H2E" name="org.ge.dctests.DC_PRTTest2" tests="3" failures="0" timestamp="20 Dec 2017 11:22:51 GMT" time="166.516" errors="1">
  <testcase name="createTeam" time="119.101" classname="org.ge.dctests.DC_PRTTest2"/>
  <testcase name="createEvent" time="45.083" classname="org.ge.dctests.DC_PRTTest2">
    <error type="org.sikuli.script.FindFailed" message="can not find P(C:/Users/212672444/Desktop/Auto_DC/AutoWorkSpace/dc-automation/resources/images/dcgantt//EventTimeImg.png) S: 0.7 in S(0)[0,0 1366x768]">
      <![CDATA[FindFailed: can not find P(C:/Users/212672444/Desktop/Auto_DC/AutoWorkSpace/dc-automation/resources/images/dcgantt//EventTimeImg.png) S: 0.7 in S(0)[0,0 1366x768]  Line 2189, in file Region.javaat org.sikuli.script.Region.handleFindFailedShowDialog(Region.java:2189)at org.sikuli.script.Region.handleFindFailed(Region.java:2134)at org.sikuli.script.Region.wait(Region.java:2546)at org.sikuli.script.Region.find(Region.java:2205)at org.sikuli.script.Region.getLocationFromTarget(Region.java:3067)at org.sikuli.script.Region.dragDrop(Region.java:3670)at org.ge.pageobjects.DC_DispatchConsolePage.dragDropWO(DC_DispatchConsolePage.java:112)at org.ge.dctests.DC_PRTTest2.createEvent(DC_PRTTest2.java:124)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </error>
  </testcase> <!-- createEvent -->
  <testcase name="createDO" time="2.332" classname="org.ge.dctests.DC_PRTTest2"/>
</testsuite> <!-- org.ge.dctests.DC_PRTTest2 -->
